core/src/main/java/hudson/slaves/NodeProvisioner.java

```diff
@@ -165,19 +167,26 @@ public NodeProvisioner(Label label, LoadStatistics loadStatistics) {
 
     /**
      * Give the {@link NodeProvisioner} a hint that now would be a good time to think about provisioning some nodes.
-     * The hint will be ignored if subjected to excessive pestering by callers.
+     * Hints are throttled to one every second.
      *
      * @since 1.415
      */
     public void suggestReviewNow() {
-        if (System.currentTimeMillis() > lastSuggestedReview + TimeUnit.SECONDS.toMillis(1)) {
+        long delay = TimeUnit.SECONDS.toMillis(1) - (System.currentTimeMillis() - lastSuggestedReview);
+        if (delay < 0) {
```

Comment:
If you have a queued review and call suggestReviewNow you can possibly call update() twice